### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'iteritems'

### ISSUE THREAD

**Issue by eMerzh** [ajslater/picopt#20]
hi, i've installed picopt on my mac via pip but it doesn't seems to work here

osx 10.11 python 3.5

did 
`pip install picopt` in my venv 
then when i launch it i have 

```
File "/dir/venv/bin/picopt", line 11, in <module>
  sys.exit(main())
File "/dir/venv/lib/python3.5/site-packages/picopt/cli.py", line 185, in main
  run(sys.argv)
File "/dir/venv/lib/python3.5/site-packages/picopt/cli.py", line 179, in run
  process_arguments(raw_arguments)
File "/dir/venv/lib/python3.5/site-packages/picopt/cli.py", line 132, in process_arguments
  Settings.update(arguments)
File "/dir/venv/lib/python3.5/site-packages/picopt/settings.py", line 39, in update
  for key, val in settings.__dict__.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

is there smth i forgot?

thanks for your great work ;-)

**Comment by msoedov** [comment]
the same

**Comment by ajslater** [comment]
fixed in 1.5.2
